### PR TITLE
Updating tools/proteinortho tool version(s)

### PR DESCRIPTION
**Pull request by planemo-autoupdate**
Hello! This is an automated update of the following tool: **tools/proteinortho**. I created this PR because I think the tool's main dependency is out of date, i.e. there is a newer version available through conda.

Please see https://github.com/planemo-autoupdate/autoupdate for more information.

### CODE REVIEW

**Comment by pmjklemm** [comment]
diamond should work fine, I will just remove the one sequence (AAAAAAA...) from the test dataset and then it is fine.

The test functions should not be removed !

**Comment by pmjklemm** [comment]
welp I cant figure out how to go into this PR and make changes, can you please paste what to clone and checkout again?

**Comment by bgruening** [comment]
@pmjklemm its this one https://github.com/planemo-autoupdate/tools-iuc.git with the branch `tools/proteinortho`

**Comment by pmjklemm** [comment]
I thought so too but i get  

```
remote: Permission to planemo-autoupdate/tools-iuc.git denied to pmjklemm.
fatal: unable to access 'https://github.com/planemo-autoupdate/tools-iuc.git/': The requested URL returned error: 403
```

git clone https://github.com/planemo-autoupdate/tools-iuc.git
gh pr checkout 3949
# changes
git add + commit + push 
(or do i need to make a push though this gh command?)

**Comment by bernt-matthias** [comment]
> diamond should work fine, I will just remove the one sequence (AAAAAAA...) from the test dataset and then it is fine.
> 
> The test functions should not be removed !

I did not remove them, but replaced them by more general ones (in the macro). This has been done because the output changed (numbers, etc) .. might this be because not the latest diamond version is used. If this is the case then I'm happy to restore the test assertions.

File: tools/proteinortho/proteinortho.xml [hunk 1]
```diff
@@ -138,124 +151,51 @@
...
-            <output name="proteinorthograph">
-                <assert_contents>
-                    <has_text text="M_317&#009;L_317&#009;"/>
-                </assert_contents>
-            </output>
+            <param name="p" value="autoblast"/>
+            <expand macro="test_outputs"/>
+            <assert_command>
+                <has_text text="--p=autoblast"/>
+            </assert_command>
```
Comment (pmjklemm) [inline tools/proteinortho/proteinortho.xml:182]: no, please do not delete all my test cases

File: tools/proteinortho/proteinortho.xml [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+            </output>
+            <output name="proteinorthograph">
+                <assert_contents>
+                    <has_line_matching expression="# file_a\tfile_b"/>
+                    <has_line_matching expression="# a\tb\tevalue_ab\tbitscore_ab\tevalue_ba\tbitscore_ba"/>
+                    <has_line_matching expression="# C|C2|E|L|M)\.fasta\tC|C2|E|L|M)\.fasta"/>
+                    <has_line_matching expression=".*(C|C2|E|L|M)_[0-9]+\t(C|C2|E|L|M)_[0-9]+.*"/>
+                </assert_contents>
+            </output>
+        </xml>
```
Comment (pmjklemm) [inline tools/proteinortho/proteinortho.xml:30]: ahh ok now i understand, it is a makro ! okok

**Comment by pmjklemm** [comment]
I cant figure out how to enter this PR with gh (never used that before) and i am tried. You can remove the C_11 from test-data/C2.fasta and diamond should be happy again.

anyway thanks !

**Comment by bernt-matthias** [comment]
I usually: 

```
git remote add planemo-autoupdate https://github.com/planemo-autoupdate/tools-iuc.git
git fetch --all
git checkout -b tools/proteinortho planemo-autoupdate/tools/proteinortho
```

If you still can not push (maybe only IUC members can) you may open a PR against the planemo-autoupdate fork or my fork. 

@bgruening any ideas about such permission problems. I have the same problem with the update-test-data slash command over here: https://github.com/galaxyproject/tools-iuc/pull/3923

**Comment by bgruening** [comment]
We should not push directly to bot accounts I assume. But we can PR against them. If people can write I think those are IUC members, but its tricky since this bot is on a different organization.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks both!

**Pull request merged**